specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2023-03-01-preview/amlfilesystem.json

```diff
@@ -0,0 +1,1108 @@
+      "post": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AmlFilesystemNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "AML file system has started the cancellation of the archive operation. Poll the AML file system's hsm.archiveStatus field to monitor the progress."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "storagecache.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "amlFilesystems_cancelArchive": {
+            "$ref": "./examples/amlFilesystems_CancelArchive.json"
+          }
+        },
+        "tags": [
+          "amlFilesystems"
+        ],
+        "description": "Cancel archiving data from the AML file system.",
+        "operationId": "amlFilesystems_CancelArchive"
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageCache/checkAmlFSSubnets": {
+      "post": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "amlFilesystemSubnetInfo",
+            "in": "body",
+            "description": "Information about the subnets to validate.",
+            "schema": {
+              "$ref": "#/definitions/AmlFilesystemSubnetInfo"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The subnets provided meet the requirements for AML file system create."
+          },
+          "400": {
+            "description": "The subnets provided do not meet the requirements for AML file system create.",
+            "schema": {
+              "$ref": "#/definitions/AmlFilesystemCheckSubnetError"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "storagecache.json#/definitions/CloudError"
+            }
+          }
```

Comment:
@brpanask 

Please note that even when this does not trigger a Linter error, this definition is not approparate for ARM.

1. Use "default" if possible.
2. If this error response is different from the "default", you can extend `CloudError`, but you should still follow [Error Response](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content)
3. Other cases is only allowed for backward compatibility concern. (but yours is new and in preview)